boards/xtensa/m5stack_atoms3lite/Kconfig.defconfig

```diff
@@ -0,0 +1,25 @@
+# M5Stack AtomS3Lite board configuration
+# Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_M5STACK_ATOMS3LITE
+
+config BOARD
+	default "m5stack_atoms3lite"
+	depends on BOARD_M5STACK_ATOMS3LITE
+
+config ENTROPY_GENERATOR
+	default y
+
+config HEAP_MEM_POOL_SIZE
```

Comment:
This config was "recently" changed to `HEAP_MEM_POOL_ADD_SIZE_BOARD`

Reply:
fixed in [bd019a9](https://github.com/zephyrproject-rtos/zephyr/commit/bd019a9cf65db130247357c3d9de3a8da0d7a504)